lib/Email/Stuffer/TestLinks.pm

```diff
@@ -0,0 +1,77 @@
+package Email::Stuffer::TestLinks;
+
+use strict;
+use warnings;
+
+our $VERSION = '0.010';
+
+use Test::Most;
+use Mojo::UserAgent;
+use Email::Stuffer;
+use Class::Method::Modifiers qw/ install_modifier /;
+
+=head1 SYNOPSIS
+
+    use Email::Stuffer::TestLinks;
+
+=head1 NAME
+
+Email::Stuffer::TestLinks - validates links in HTML emails sent by Email::Stuffer>send_or_die()
+
+=head1 DESCRIPTION
+
+When this module is included in a test, it parse HTML links in every email sent through check Email::Stuffer.
+Only html links (<a href="xyz"...) are checked. Each link must be valid resource and the returned page title must not contain 'error' or 'not found'.
```

Comment:
Should "HTML" be capitalised here, as it was on the previous line?

Reply:
Thanks for the feedback, I rewrote the description